maintainers/maintainer-list.nix

```diff
@@ -15796,6 +15796,12 @@
     github = "plabadens";
     githubId = 4303706;
     keys = [ { fingerprint = "B00F E582 FD3F 0732 EA48  3937 F558 14E4 D687 4375"; } ];
+  };
+    pladypus = {
```

Comment:
`pladypus` is out of indentation. I suggest you to use `nixfmt` on this file.

Using [nixfmt-rfc-style](https://search.nixos.org/packages?query=nixfmt) package from unstable.

> nix shell github:NixOS/nixpkgs/nixos-unstable#nixfmt-rfc-style

> nixfmt maintainers/maintainer-list.nix